CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
+# 2.0.0 
+
+## Enhancements
+
+- Added DeleteConsumerGroupOffset to AdminClient.
```

Comment:
This is for https://github.com/confluentinc/confluent-kafka-dotnet/commit/fa7626c49fc5955a82db276d199b73361a739e16 where I missed updating the changelog.